cdap-ui/app/features/adapters/etlapps.less

```diff
@@ -193,7 +191,7 @@ body.theme-cdap.state-adapters-create {
         .modal-btns {
           margin-right: -15px;
           text-align: right;
-
+          height: 60px;
```

Comment:
We might not need a height here. The individual button widths need to be managed with media queries.